builder/amazon/common/state.go

```diff
@@ -66,6 +66,19 @@ func WaitUntilAMIAvailable(ctx aws.Context, conn ec2iface.EC2API, imageId string
 	return err
 }
 
+func WaitUntilInstanceRunning(ctx aws.Context, conn *ec2.EC2, instanceId string) error {
+
+	instanceInput := ec2.DescribeInstancesInput{
+		InstanceIds: []*string{&instanceId},
+	}
+
+	err := conn.WaitUntilInstanceRunningWithContext(
```

Comment:
Oh super nice, I wasn't aware of this one !

Reply:
Nice! Seeing as we don’t actually use err here we can just return with the call to `conn.WaitUntilInstanceRunningWithContext(...`

Reply:
@nywilken I'm following the convention of other existing WaitUntil* methods in state.go

Reply:
@fly1028 thanks I only looked at the diff. But this is good to go I just have not had a chance to merge.